src/bitcoin-cli.cpp

```diff
@@ -46,6 +46,7 @@ std::string HelpMessageCli()
     strUsage += HelpMessageOpt("-rpcpassword=<pw>", _("Password for JSON-RPC connections"));
     strUsage += HelpMessageOpt("-rpcclienttimeout=<n>", strprintf(_("Timeout in seconds during HTTP requests, or 0 for no timeout. (default: %d)"), DEFAULT_HTTP_CLIENT_TIMEOUT));
     strUsage += HelpMessageOpt("-stdin", _("Read extra arguments from standard input, one per line until EOF/Ctrl-D (recommended for sensitive information such as passphrases)"));
+    strUsage += HelpMessageOpt("-wallet=<walletname>", _("Defines which wallet should be used if a wallet related command gets executed (default: default-wallet if only one present, non-optional if running multiple wallets)"));
```

Comment:
Thread https://github.com/bitcoin/bitcoin/pull/10650#discussion_r127314126.

I see, yeah. I guess `-wallet` won't work unless bitcoin-cli is changed to explicitly ignore `-wallet` values that come from the conf file, which might not be a bad idea, but I'm not advocating for.

Interestingly, I guess if if `bitcoin-cli` is changed to accept `-usewallet` and support for node calls on the wallet endpoint is added, users will be able to specify a "default wallet" in their configs.